lib/cli/handlers/status.js

```diff
@@ -0,0 +1,51 @@
+const path = require('path');
+const chalk = require('chalk');
+
+const { logRed, logGreen, getComposeFileOrFlagsHandler } = require('./../methods');
+const { Polygon } = require('../../services/polygon');
+const { Terraform } = require('./../../../lib/services/terraform');
+const { prepareArgsForPolygonOps } = require('./common');
+
+async function status({ fileMode = false, __fileDirname = '', awsProfile, sshPublicKey: _sshPublicKey,
+    orbsAddress, orbsPrivateKey, region, nodeSize, nodeCount, publicIp,
+    cachePath: _catchPath, name, bootstrapUrl, backend = false, ephemeralStorage = false,
+    configPath: _configPath, chainVersion, ethereumEndpoint,
+    ethereumTopologyContractAddress, incomingSshCidrBlocks,
+    sslCertificatePath, sslPrivateKeyPath,
+}) {
+    const { cloud, keys } = prepareArgsForPolygonOps({
+        fileMode, __fileDirname, _configPath, _sshPublicKey, _catchPath,
+        chainVersion, ethereumTopologyContractAddress, sslCertificatePath, sslPrivateKeyPath,
+        orbsAddress, orbsPrivateKey, awsProfile, incomingSshCidrBlocks, ethereumEndpoint,
+        region, backend, nodeSize, nodeCount, bootstrapUrl, name, ephemeralStorage,
+    });
+
+    cloud.ip = publicIp;
+
+    const terraformAdapter = new Terraform();
+    terraformAdapter.setCachePath(path.join(process.cwd(), '_terraform'));
+
+    const polygon = new Polygon({ terraformAdapter });
+    polygon.setTerraformCachePath(cloud.cachePath);
```

Comment:
Why is it set from the outside and not provided by `terraformAdapter`?

Reply:
The Polygon service needs to know the cache path as well since it's basically the working directory. Ever since we started inserting stuff into that directory for non Terraform usage as well we also need that path set in our own service to be able to easily write files for the project state.

This is introduced even before this PR for the sake of the transactional create and destroy markers and other stuff as well.

Reply:
Polygon doesn't exist without the adapter, so polygon cache path should be taken from the adapter with a direct function call.

Reply:
You mean perform something like `terraformAdapter.getWorkingDir()` or some sort?